hub/policy/src/test/java/uk/gov/ida/hub/policy/logging/HubEventLoggerTest.java

```diff
@@ -156,25 +168,27 @@ public void logIdpAuthnSucceededEvent_shouldContainLevelsOfAssuranceInDetailsFie
             PERSISTENT_ID,
             REQUEST_ID,
             MINIMUM_LEVEL_OF_ASSURANCE,
-            REQUIRED_LEVEL_OF_ASSURANCE,
+            MAXIMUM_LEVEL_OF_ASSURANCE,
+            PREFERRED_LEVEL_OF_ASSURANCE,
             PROVIDED_LEVEL_OF_ASSURANCE,
             Optional.ofNullable(PRINCIPAL_IP_ADDRESS_SEEN_BY_IDP),
             PRINCIPAL_IP_ADDRESS_SEEN_BY_HUB,
             ANALYTICS_SESSION_ID,
             JOURNEY_TYPE
         );
 
-        final EventSinkHubEvent expectedEvent = createExpectedEventSinkHubEvent(Map.of(
-                idp_entity_id, IDP_ENTITY_ID,
-                pid, PERSISTENT_ID.getNameId(),
-                minimum_level_of_assurance, MINIMUM_LEVEL_OF_ASSURANCE.name(),
-                required_level_of_assurance, REQUIRED_LEVEL_OF_ASSURANCE.name(),
-                provided_level_of_assurance, PROVIDED_LEVEL_OF_ASSURANCE.name(),
-                principal_ip_address_as_seen_by_idp, PRINCIPAL_IP_ADDRESS_SEEN_BY_IDP,
-                principal_ip_address_as_seen_by_hub, PRINCIPAL_IP_ADDRESS_SEEN_BY_HUB,
-                session_event_type, IDP_AUTHN_SUCCEEDED,
-                analytics_session_id, ANALYTICS_SESSION_ID,
-                journey_type, JOURNEY_TYPE));
+        final EventSinkHubEvent expectedEvent = createExpectedEventSinkHubEvent(Map.ofEntries(
```

Comment:
NB. `Map.of` supports up to 10 parameters and now we have 11 so I had to switch it to `Map.ofEntries`.